pkg/kudoctl/cmd/init.go

```diff
@@ -105,6 +108,7 @@ func newInitCmd(fs afero.Fs, out io.Writer) *cobra.Command {
 	f.Int64Var(&i.timeout, "wait-timeout", 300, "Wait timeout to be used")
 	f.StringVar(&i.webhooks, "webhook", "", "List of webhooks to install separated by commas (One of: InstanceValidation)")
 	f.StringVarP(&i.serviceAccount, "service-account", "", "", "Override for the default serviceAccount kudo-manager")
+	f.BoolVar(&i.selfSignedWebhookCA, "self-signed-webhook-ca-for-testing-only", false, "Use self-signed CA bundle (for testing only) for the webhooks")
```

Comment:
I'm not a fan of the "testing-only" suffix... do we need to add "-ga" or "-prod" to everything else?   I would much rather have the shorter flag name.   IF you want to double check, lets add a confirmation requiring a Y/n if flag detected.  I would rather just document it, it's purpose and consequences.  It is up to users as to when they will use it or not.

Reply:
I don't mind making it shorter at all, however, there were a number of concerns that users will end up using it in production. Hence the long and descriptive name making it clear that it shouldn't be used in prod.

Reply:
if you don't mind.. who does?  and what are the "number" of concerns... there seems to be only one...